packages/pages/src/dev/server/autoInit.ts

```diff
@@ -5,17 +5,19 @@ import chalk from "chalk";
 import { parseYextrcContents } from "../../util/yextrcContents.js";
 import { logErrorAndExit } from "../../util/logError.js";
 
-export const autoYextInit = async () => {
+export const autoYextInit = async (scope: string | undefined) => {
   if (!fs.existsSync(".yextrc")) {
-    await autoYextInitWithoutYextrc();
+    await autoYextInitUpdateYextrc(scope);
```

Comment:
I think I prefer the previous name. Though really we should change the flow. If the file doesn't exist then we create it. After that we try to retrieve the data. If it's not there, then we do the prompt and add it.

Reply:
Adjusted flow.